.github/workflows/macos-build.yml

```diff
@@ -578,6 +787,11 @@ jobs:
         unzip -:o dist.zip
         unzip -:o chromedriver.zip
         unzip -:o mksnapshot.zip
+    - name: Import & Trust Self-Signed Codesigning Cert on MacOS
+      run: |
+        sudo security authorizationdb write com.apple.trust-settings.admin allow
+        cd src/electron
+        ./script/codesign/generate-identity.sh
```

Comment:
What is this needed for?

Reply:
@jkleinsc this enables codesigning tests - https://github.com/electron/electron/blob/c670e38b4b14d0286e0ae04cb7f63987beb5e044/.circleci/config/base.yml#L562-L570